docs/getting-started/security-ui.asciidoc

```diff
@@ -22,204 +23,69 @@ image::images/search-bar.png[]
-
-[float]
-[[getting-started-page]]
-=== Get started page
-
-The Get started page guides you to integrations that you can use to monitor your hosts and ingest data. Other {security-app} pages that show event data (for example, the Hosts and Network pages) display the get started prompt until event data has been ingested.
-
-image::images/getting-started-pg.png[Shows the Get started page]
-
-[float]
-[[overview-ui]]
-=== Overview dashboard
-
-The Overview dashboard provides a high-level snapshot of detections, external alerts, and event trends. It can help you assess overall system health and find anomalies that may require further investigation. Refer to <<overview-dashboard, Overview dashboard>> for more information.
-
-image::images/overview-pg.png[Overview dashboard]
-
-[float]
-[[detection-response-dashboard-ui]]
-=== Detection & Response dashboard
-
-The Detection & Response dashboard provides focused visibility into the day-to-day operations of your security environment. It helps security operations managers and analysts quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with alerts. Refer to <<detection-response-dashboard, Detection & Response dashboard>> for more information.
-
-[role="screenshot"]
-image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
-
-[float]
-[[detection-engine-ui]]
-=== Alerts page
-
-The Alerts page allows you to view and manage all alerts to monitor activity within your network. Refer to <<detection-engine-overview, Detections and Alerts>> for more information.
+<<detection-engine-overview, *Alerts*>>: View and manage alerts to monitor activity within your network.
 
 [role="screenshot"]
 image::detections/images/alert-page.png[]
 
-[float]
-=== Findings page
-
-The Findings page provides an overview of your Kubernetes infrastructure's performance on a range of security benchmarks. Refer to <<kspm, Kubernetes security posture management>>.
+<<findings-page, *Findings*>>: Review your Kubernetes infrastructure’s performance on a range of security benchmarks.
 
 [role="screenshot"]
 image::cloud-native-security/images/findings-page.png[The Findings page]
 
-[float]
-[[rules-page]]
-=== Rules page
-
-The Rules page allows you to view and manage all detection rules. Refer to <<rules-ui-management, Manage detection rules>> for more information about prebuilt and custom rules.
+<<timelines-ui, *Timelines*>>: Investigate alerts and complex threats, such as lateral movement of malware across hosts in your network. Timelines are responsive and allow you to share your findings among other team members.
 
 [role="screenshot"]
-image::detections/images/all-rules.png[Shows the Rules page]
+image::images/timeline-ui.png[Shows the Timeline page]
 
-[float]
-[[exceptions-page]]
-=== Exception lists page
+TIP: Select the collapsable *Timeline* button at the bottom of the {security-app} to start an investigation.
```

Comment:
This isn't a correction, but a note that we _might_ need to update this line in 8.5. At the start of this week, I built an ESTEC deployment using the 8.5-SNAPSHOT and upon reviewing it, noticed that the Timeline component was already expanded at the bottom of Security app. I'm seeing something different on the endpoint test server (which is also running a snapshot of 8.5.0) so that change might've been reverted. Anyways, this is something I plan to verify for 8.5.0.

Reply:
Verified with @michaelolo24 that this is actually a bug and Timeline has not been redesigned in 8.5. Ignore this comment!